src/Runner.Worker/JobExtension.cs

```diff
@@ -316,6 +316,23 @@ public async Task<List<IStep>> InitializeJob(IExecutionContext jobContext, Pipel
                         }
                     }
 
+                    if (message.Variables.TryGetValue("system.workflowFileFullPath", out VariableValue workflowFileFullPath))
+                    {
+                        if (message.ContextData.TryGetValue("inputs", out var pipelineContextData)) {
```

Comment:
do we want to put these into a `##[group]` so it can collapse in case there are lots of inputs?